src/settings.zig

```diff
@@ -79,6 +81,10 @@ pub fn init() void {
 		}
 	}
 
+	if(cpuThreads == null) {
+		cpuThreads = @max(1, (std.Thread.getCpuCount() catch 4) -| 1);
```

Comment:
Setting the value here, means that it will be saved as such.
So if I were to play Cubyz on a 4 thread CPU, it will store the value 3. If I then move my settings.json to a different computer with more threads it will still only use 3 of them and the user has no clue why this is happening.

Instead I would suggest to leave it null here and do this calculation in `main.zig`.

Reply:
Ah. Setting it to null still leaves it for the user to find. Will do.